app/webpacker/src/stylesheets/_help_popup.scss

```diff
@@ -0,0 +1,32 @@
+#help_popup {
+  background-color: #ffffff;
+  border: 1px #333 solid;
```

Comment:
Again - would this be better using a standard colour? Possibly one to ask matt

`govuk-colour("mid-grey")` maybe?